srcpkgs/obs-v4l2sink/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'obs-v4l2sink'
+pkgname=obs-v4l2sink
+version=3.5
+revision=1
+archs="x86_64"
+depends="obs v4l2loopback"
+short_desc="OBS plugin that provides output capabilities to a v4L2 device"
+maintainer="xSlendiX <radu.ungureanu@techie.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/CatxFish/obs-v4l2sink"
+
+do_install() {
+	mkdir -r /usr/lib/obs-plugins
+	mkdir -r /usr/share/obs/obs-plugins/v4l2sink/locale
+
+	vinstall v4l2sink.so 755 /usr/lib/obs-plugins
+	vinstall *.ini 644 /usr/lib/obs-plugins
```

Comment:
Install paths shouldn't start with `/`. Installing multiple files should be done with a `for file in *.ini` loop.

Reply:
for file in *.ini doesnt work. it takes * literally